Bioinfo_joint/HW4_modules/natls.py

```diff
@@ -0,0 +1,110 @@
+def dict_comp(nucleotide: str) -> str:
+    complement = dict(
+        [
+            ("a", "t"),
+            ("A", "T"),
+            ("c", "g"),
+            ("C", "G"),
+            ("g", "c"),
+            ("G", "C"),
+            ("t", "a"),
+            ("T", "A"),
+            ("u", "a"),
+            ("U", "A"),
+            ("0", "0"),
+        ]
+    )
+    return complement[nucleotide]
+
+
+def dict_trans(nucleotide: str) -> str:
+    transcript = dict(
+        [
+            ("a", "a"),
+            ("A", "A"),
+            ("c", "c"),
+            ("C", "C"),
+            ("g", "g"),
+            ("G", "G"),
+            ("t", "u"),
+            ("T", "U"),
+            ("0", "0"),
+        ]
+    )
+    return transcript[nucleotide]
+
+
+def is_dnasubset(seq: str) -> bool:
+    if set(seq).issubset({"A", "T", "G", "C", "a", "t", "g", "c"}):
+        return True
+
+
+def is_rnasubset(seq: str) -> bool:
+    if set(seq).issubset({"A", "U", "G", "C", "a", "u", "g", "c"}):
+        return True

```

Comment:
Аналогично функции `is_dnasubset`
```suggestion
    return set(seq).issubset({"A", "U", "G", "C", "a", "u", "g", "c"})
```